src/Service.Tests/dab-config.MsSql.json

```diff
@@ -2443,6 +2443,33 @@
         "bkname": "name"
       }
     },
+    "BooksSold": {
+      "source": {
+        "object": "books_sold",
+        "type": "table"
+      },
+      "graphql": {
+        "enabled": true,
+        "type": {
+          "singular": "books_sold",
+          "plural": "books_sold"
+        }
+      },
+      "rest": {
+        "enabled": true,
+        "methods": []
+      },
+      "permissions": [
+        {
+          "role": "anonymous",
+          "actions": [
+            {
+              "action": "*"
```

Comment:
This doesn't look right. If you want all actions, it is a one liner: `"actions": ["*"]`

Reply:
I think this is how the config is generated using the config generating command. I just copy pasted from what it actually looks like when config is generated. And that's why you'll see there are other instances of this same format within this same file.

Reply:
Yes, this is how the new config system creates it. The one line is accepted as well while deserializing but when we serialize this is the new way to write it.